klaytnetl/jobs/exporters/raw_block_group_item_exporter.py

```diff
@@ -37,8 +37,8 @@
     "extra_data",
     # 'gas_limit',  # Does not supported by klay_getBlockWithConsensusInfoByNumber
     "gas_used",
-    "timestamp",
-    "unix_timestamp",
+    "block_timestamp",
+    "block_unix_timestamp",
```

Comment:
Was this unchanged?

Reply:
Yes :( I changed the name to `block_timestamp` and `block_unix_timestamp` for consistency